rigs/heft-web-rig/shared/webpack-base.config.js

```diff
@@ -0,0 +1,270 @@
+              // https://www.npmjs.com/package/css-loader
+              loader: require.resolve('css-loader'),
+              options: {
+                // 0 => no loaders (default);
+                // 1 => postcss-loader;
+                // 2 => postcss-loader, sass-loader
+                importLoaders: 2,
+
+                // Enable CSS modules:  https://github.com/css-modules/css-modules
+                modules: {
+                  // The "auto" setting has a confusing design:
+                  // - "false" disables CSS modules, i.e. ":local" and ":global" selectors can't be used at all
+                  // - "true" means magically disable CSS modules if the file extension isn't like ".module.css"
+                  //   or ".module.scss"
+                  // - a lambda disables CSS modules only if the lambda returns false; the function parameter is
+                  //   the resource path
+                  // - a RegExp disables CSS modules only if the resource path does not match the RegExp
+                  //
+                  // NOTE: Counterintuitively, if you instead set "modules=true" then CSS modules are enabled
+                  //       without magic, equivalent to "auto: () => true" instead of "auto: true"
+                  //
+                  // DEFAULT: "true" (i.e. path based magic)
+                  auto: (resourcePath) => {
+                    // Enable CSS modules...
+                    const useCssModules =
+                      // ...UNLESS the filename opts out using a file extension like "filename.global.scss"
+                      !/\.global\.\w+$/i.test(resourcePath) &&
+                      // ...UNLESS this is a .css file.
+                      !/\.css$/i.test(resourcePath);
+                    return useCssModules;
+                  },
+
+                  // This setting has no effect unless CSS modules is enabled. Possible values:
+                  // - "local": global CSS by default, overridable using the ":local" selector
+                  // - "global": local CSS by default, overridable using the ":global" selector
+                  // - "pure": requires selectors to contain at least one local class or id
+                  // - a lambda that returns the mode string; the function parameter is the resource path
+                  //
+                  // DEFAULT: "local"
+                  mode: 'local',
+
+                  // Set this to true if you want to be able to reference the global declarations using import statements
+                  // similar to local CSS modules
+                  //
+                  // DEFAULT: false
+                  // exportGlobals: true,
+
+                  // Provide a recognizable class/module names for developers
+                  //
+                  // DEFAULT: "[hash:base64]"
+                  localIdentName: production ? '[hash:base64]' : '[local]__[hash:base64:5]'
+                },
+
+                sourceMap: !production
```

Comment:
One weirdness of the above approach is that global .css files get sourcemaps injected, which you can see in the Chrome F12 debugger.  (Does that even make sense when CSS modules are not being used?)

However because we are using 'mini-css-extract-plugin', the problem disappears when the CSS is extracted into a separate bundle.